packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -239,15 +239,11 @@ export class TableQuadrantStack extends AbstractComponent<ITableQuadrantStackPro
     public constructor(props: ITableQuadrantStackProps, context?: any) {
         super(props, context);
 
-        // a few points here:
-        // - we throttle onScroll/onWheel callbacks to making scrolling look more fluid.
-        // - we declare throttled functions on the component instance, since they're stateful.
-        // - "wheel"-ing triggers super-fluid onScroll behavior by default, but relying on that
-        //   causes sync'd quadrants to lag behind. thus, we preventDefault for onWheel and instead
-        //   manually update all relevant quadrants using event.delta{X,Y} later, in the callback.
-        //   this keeps every sync'd quadrant visually aligned in each animation frame.
+        // callbacks trigger too frequently unless we throttle scroll and wheel
+        // events. declare these functions on the component instance since
+        // they're stateful.
         this.throttledHandleMainQuadrantScroll = CoreUtils.throttleReactEventCallback(this.handleMainQuadrantScroll);
-        this.throttledHandleWheel = CoreUtils.throttleReactEventCallback(this.handleWheel, { preventDefault: true });
```

Comment:
But now:
![scroll-pull](https://user-images.githubusercontent.com/569364/31295509-a3a84d12-aa93-11e7-81ce-c1704d7f3287.gif)

Reply:
I'm good with this; it's essentially expected browser behavior, and the benefits of not having your scroll swallowed are worth it (see the "Features" page to feel that benefit in action).